src/bin/list_store.rs

```diff
@@ -0,0 +1,299 @@
+// "List Store" gtk3 demo ported to gtk-rs by Ömer Sinan Ağacan <omeragacan@gmail.com>
```

Comment:
Please use same header as everywhere else (if we have one?). Git blame is there to know who wrote the code. ;)

Reply:
This looks fine to me. I don't think we have a standard header for examples. Lots of examples don't have any headers.

Reply:
If all people were starting to add their names in the header, it wouldn't be sustainable much time. Again, there are git logs. The headers in those examples are only supposed to describe shortly what it is about.